src/pages/index.js

```diff
@@ -1,8 +1,74 @@
 import React from 'react'
 import { Helmet } from 'react-helmet'
 import s from 'styled-components'
+//import MDEditor from '@uiw/react-md-editor';
+import InputField from '../components/InputField.js'
+import { Navbar, Button } from 'react-bootstrap'
 
```

Comment:
import can be categorized into two types: external imports (imports from packages) and internal imports (imports from files that we wrote). convention is to put external imports at top, followed by a newline, and the internal imports. so basically
```
import React from 'react'
import { Helmet } from 'react-helmet'
import s from 'styled-components'
import { Navbar, Button } from 'react-bootstrap'

import InputField from '../components/InputField'
```
Notice that we **do not** need to specify `.js` extension for `InputField` because internal imports by default first look at if there is a file with the `.js` extension